### DISCUSSION TITLE
Support for YI 1080p Home 18US

### DISCUSSION THREAD

**Discussion by Piecho2** [roleoroleo/yi-hack-Allwinner-v2#774] [Ideas]
Hello,
Managed to flash Yi-Hack v5 v0.38 on YI 1080p Home 18US. Any chance supporting this this serial series in the future? Warm thanks for your community involvement!

**Comment by roleoroleo** [comment]
It's a different platform.
Not supported by this hack.
But the content of the hack is not so different: RTSP. ONVIF,we gui, etc...

**Comment by Piecho2** [comment]
Is there any way that I can share logs/output from my device so it could be possibly supported in the future releases?

**Reply by roleoroleo** [reply]
No, I will not develop a new hack for this platform.
v5 is a great hack.
